ToasterMessage.js

```diff
@@ -221,6 +221,44 @@ define(["dcl/dcl",
 		}.bind(this));
 	};
 
+	var PauseTimerOnHover = function (element) {
+
+		var hovering = false;
+		function _pointerOverHandler() {
+			console.log("hover event");
+			if (!hovering) {
+				hovering = true;
+				element._timer.pause();
+			}
+		}
+
+		function _pointerLeaveHandler() {
+			if (hovering) {
+				hovering = false;
+				element._timer.resume();
+			}
+		}
+
+		this.isEnabled = false;
```

Comment:
Yes, as the feature can be enabled/disabled, I think it could be useful to the user to know that value.